spatial/workers/unreal/spatialos.UnrealWorker.worker.json

```diff
@@ -66,6 +66,8 @@
       "arguments": [
         "${IMPROBABLE_WORKER_ID}",
         "${IMPROBABLE_LOG_FILE}",
+        "+deploymentName",
```

Comment:
What's the reasoning behind this change? Should we change our default worker templates also?

Reply:
My custom `StartWorker.sh` that overrides the original in the assemlby uses this `deploymentName` parameter to extract any embedded map specification (for deployment names of the format `ANYTHING_map_MAPNAME_ANYTHINGELSE`). This `StartWorker.sh` override is only present in the `ShooterGameBuildkite` repo, so passing this extra parameter would not have any effect in other templates.

Reply:
Cool - thanks for the info